src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -37,6 +37,15 @@ public static class CLIOptions
             JsonKey = "log-file"
         };
 
+        public static readonly CLIOption<bool> DevMode = new CLIOption<bool>
+        {
+            ArgumentName = "--dev-mode",
+            ArgumentShortName = "-dm <enable>",
+            ArgumentDescription = "Set to true to activate dev mode to ease debugging of Stryker.Net| Options [true, false (default)]",
+            DefaultValue = false,
+            JsonKey = "safe-mode"
```

Comment:
Is it called dev-mode or safe mode? Both are used currently. For example the unit test also uses --safe-mode

Reply:
actually, it's dev-mode. Safe mode came from previous conversations.  need to fix that.